lighthouse-core/scoring.js

```diff
@@ -18,17 +18,22 @@ const clampTo2Decimals = val => Math.round(val * 100) / 100;
 class ReportScoring {
   /**
    * Computes the weighted-average of the score of the list of items.
-   * @param {Array<{score: number, weight: number}>} items
-   * @return {number}
+   * @param {Array<{score: number|null, weight: number}>} items
+   * @return {number|null}
    */
   static arithmeticMean(items) {
+    // If there is 1 null score, return a null average
+    if (items.some(item => item.score === null && item.weight !== 0)) return null;
```

Comment:
wdyt about an early filter in this method to get rid of the weight===0 items. then we don't need the check in here PLUS L32.
ndb, tho

Reply:
yeah sure sg 👍